.github/workflows/ssh-session.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         if: ${{ github.event.inputs.runs-on == 'ubuntu-latest'  && github.event.inputs.extend-space == 'true' }}
 
       - name: Install nix
-        uses: cachix/install-nix-action@v18
+        uses: cachix/install-nix-action@v20
```

Comment:
The latest is v22, should we update to that?

Reply:
it's just what we use on develop